client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -226,8 +226,8 @@ public class ClientOptions
     @Option(names = "--path", paramLabel = "<catalog.schema>", description = "Default SQL path", arity = "0..*")
     public List<String> path = ImmutableList.of();
 
-    @Option(names = {"-f", "--file"}, paramLabel = "<file>", description = "Execute statements from file and exit")
-    public String file;
+    @Option(names = {"-f", "--file"}, paramLabel = "<file>", description = "Execute statements from file and exit (can be used multiple times)")
```

Comment:
`(can be used multiple times)` is not needed. I'd change paramLabel to `<files>` and description `Execute statements from files and exit`